backend/internal/server/apis/stats/shotsongoal.go

```diff
@@ -0,0 +1,41 @@
+package stats
+
+import (
+	"github.com/jak103/powerplay/internal/server/apis"
+	"github.com/jak103/powerplay/internal/utils/locals"
+	"github.com/jak103/powerplay/internal/utils/responder"
+	"github.com/gofiber/fiber/v2"
+	"github.com/jak103/powerplay/internal/db"
+	"github.com/jak103/powerplay/internal/server/services/auth"
+	"github.com/jak103/powerplay/internal/models"
+)
+
+func init() {
+	apis.RegisterHandler(fiber.MethodPost, "/shotsongoal", auth.Public, postShotsOnGoalHandler)
+	// Todo add register handler for GET
+}
+
+func postShotsOnGoalHandler(c *fiber.Ctx) error {
+	log := locals.Logger(c)
+	log.Info("Handling creating new shot on goal")
```

Comment:
We don't need this first log.Info, Dr. Christensen said the logger does this automatically